trampoline_v2.sh

```diff
@@ -64,6 +64,10 @@ else
   readonly IO_COLOR_RESET=""
 fi
 
+function fn_exists {
```

Comment:
nit: this abbreviation seems unnecessary

Reply:
sure